### DISCUSSION TITLE
GC9A01 and Arduino Mega

### DISCUSSION THREAD

**Discussion by czelek** [moononournation/Arduino_GFX#256] [General]
Hello, first of all thanks for this lib! Can I use it with GC9A01 and Arduino Mega?
So far I can see only backlight.

I tried combinations: 
First one: 
> VCC -> 3.3V
> GND -> GND
> DIN -> D11
> CLK -> D13
> CS -> D10
> DC -> D7
> RST -> D8
> BL -> D9

Second one: 
> Vin  -> Vcc
> GND -> GND
> GPIO 9 -> CS
> GPIO 7 -> RST
> GPIO 8 -> DC
> GPIO 11 -> DIN
> GPIO 13 ->CLK
> GPIO 6 -> BL

I tried with and without 2.2 resistors.
I have some warnings during compilation also: 

> GFX_Library_for_Arduino/src/Arduino_GFX.h:49:0: warning: "PINK" redefined
> define PINK 0xFC18        ///< 255, 130, 198

Thank you!

**Comment by moononournation** [comment]
same as ili9341, you need some resistors

**Reply by czelek** [reply]
I used 2.2k. For 6 pins (not for vcc and gnd). Is it working with Arduino Mega? Could you write which setting of wires is Ok? Thank you!

**Reply by moononournation** [reply]
I am using 3.3k: https://www.instructables.com/ArduinoGFX/

**Comment by czelek** [comment]
Ok. I will try that! Thank you

**Reply by czelek** [reply]
Unfortunately it isn’t working. With 3.3 k rasistors i can see only backlight same as without it. Could you write what wires connections i should use?
